dev-cpp/valijson/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
We can't add new packages as m-n: https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602.

Also, I know you've been avoiding it (😆) but I think you're the best person to handle sysdig & friends, even if it's a difficult package to handle sometimes, as you actually use it.

Reply:
Added myself as proxy-maintainer.